src/collective/easyform/serializer.py

```diff
@@ -37,8 +38,8 @@
 class SerializeToJson(DXContentToJson):
     def __call__(self, version=None, include_items=True):
         result = super(SerializeToJson, self).__call__(version, include_items)
-        self.serializeSavedData(result)
-
+        if api.user.has_permission('collective.easyform.DownloadSavedInput', obj=self.context):
```

Comment:
Please import this from `config.DOWNLOAD_SAVED_PERMISSION`.

Reply:
done